sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -265,7 +266,12 @@ def get_acc_info(row) -> Tuple[Optional[str], float]:
         def get_vcpus(row) -> float:
             if not np.isnan(row['vCPU']):
                 return float(row['vCPU'])
-            return float(row['VCpuInfo']['DefaultVCpus'])
+            try:
+                return float(row['VCpuInfo']['DefaultVCpus'])
+            except Exception as e:
```

Comment:
Do we need an `# pylint: disable=broad-except` here? Seems like the pylint test is passed

Reply:
Good catch! Thanks!